src/main/resources/resources/logisim/settings.properties

```diff
@@ -1 +1 @@
-locales = de el en es fr pt ru it nl pl
+locales = de el en es fr pt ru it nl pl ja_JP
```

Comment:
I guess `ja_JP` should be changed to `ja` to match the pattern of the other languages.